lms/envs/common.py

```diff
@@ -1858,6 +1858,8 @@
     'provider.oauth2',
     'oauth2_provider',
 
+    'third_party_auth',
```

Comment:
I don't think having this in the list has any effect besides Django running it checks on it on startup.